cmd/juju/common/controller.go

```diff
@@ -129,6 +129,7 @@ func WaitForAgentInitialisation(
 			strings.HasSuffix(errorMessage, "connection refused"),
 			strings.HasSuffix(errorMessage, "connection is shut down"),
 			strings.HasSuffix(errorMessage, "i/o timeout"),
+			strings.HasSuffix(errorMessage, "deadline exceeded"),
 			strings.HasSuffix(errorMessage, "no api connection available"),
 			strings.Contains(errorMessage, "spaces are still being discovered"):
```

Comment:
As an aside, this might be vestigial and no longer required. I can't see any non-test code that actually emits this.